sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared/base_client.py

```diff
@@ -218,7 +220,7 @@ def _create_pipeline(self, credential, **kwargs):
         # type: (Any, **Any) -> Tuple[Configuration, Pipeline]
         self._credential_policy = None
         if hasattr(credential, "get_token"):
-            self._credential_policy = BearerTokenCredentialPolicy(credential, STORAGE_OAUTH_SCOPE)
+            self._credential_policy = StorageBearerTokenCredentialPolicy(credential)
```

Comment:
Let's not include this bearer token stuff in any other package than Blob.

Don't forget to clean up import and async too :) (all packages)